src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -587,4 +651,21 @@ public partial interface INavigationInterception
         public bool IsNavigationIntercepted { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
         public string Location { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
     }
+    public partial class Router : Microsoft.AspNetCore.Components.IComponent, Microsoft.AspNetCore.Components.IHandleAfterRender, System.IDisposable
```

Comment:
Not seeing where this is removed from. Was the manual file incomplete?

Reply:
Yes, that's the issue I'm solving.

Reply:
I don't really understand how this worked before then (as in, how VS had the right tag helpers for `Router` and its params), but it definitely did.

Reply:
Because it broke a few days ago. All of our tests use the non-ref assemblies.